dataland-frontend/src/components/general/DatasetsTabMenu.vue

```diff
@@ -23,6 +23,7 @@ import TabPanel from 'primevue/tabpanel';
 import { checkIfUserHasRole, KEYCLOAK_ROLE_ADMIN, KEYCLOAK_ROLE_REVIEWER } from '@/utils/KeycloakUtils';
 import type Keycloak from 'keycloak-js';
 import { CompanyRole, type CompanyRoleAssignment } from '@clients/communitymanager';
+import router from '@/router';
 
```

Comment:
Der "data" Block ist nicht type-safe (benutzt `as Tab[]`).
Man kann ihn type-safe machen:

`data(): { tabs: Tab[] } {
    return {
    tabs: [
      {
        label: 'COMPANIES',
        route: '/companies',
        isVisible: true,
      },
      {
        label: 'MY DATASETS',
        route: '/datasets',
        isVisible: true,
      },
      {
        label: 'QA',
        route: '/qualityassurance',
        isVisible: false,
      },
      {
        label: 'MY DATA REQUESTS',
        route: '/requests',
        isVisible: true,
      },
      {
        label: 'DATA REQUESTS FOR MY COMPANIES',
        route: '/companyrequests',
        isVisible: false,
      },
      {
        label: 'ALL DATA REQUESTS',
        route: '/requestoverview',
        isVisible: false,
      },
    ],
    }
  },`